fabric-networking-v0/src/main/java/net/fabricmc/fabric/mixin/networking/MixinClientPlayNetworkHandler.java

```diff
@@ -46,18 +46,13 @@
 public abstract class MixinClientPlayNetworkHandler implements PacketContext {
 	@Shadow
 	private MinecraftClient client;
-
 	@Shadow
 	public abstract void sendPacket(Packet<?> var1);
 
 	@Inject(at = @At("RETURN"), method = "onGameJoin")
 	public void onGameJoin(GameJoinS2CPacket packet, CallbackInfo info) {
 		Optional<Packet<?>> optionalPacket = PacketRegistryImpl.createInitialRegisterPacket(ClientSidePacketRegistry.INSTANCE);
-
-		//noinspection OptionalIsPresent
-		if (optionalPacket.isPresent()) {
-			sendPacket(optionalPacket.get());
-		}
+		optionalPacket.ifPresent(this::sendPacket);
```

Comment:
don't go functional and kill performacne if you don't need to

Reply:
It's one lambda. I don't think we are sacrificing too much performance here. Plus I'd argue unwrapping an optional like before is gonna have the same impact already.